exercises/eliuds-eggs/canonical-data.json

```
@@ -0,0 +1,50 @@
+{
+  "exercise": "eliuds-eggs",
+  "cases": [
+    {
+      "uuid": "559e789d-07d1-4422-9004-3b699f83bca3",
+      "description": "pop-count of 0",
```

Maybe the description could focus a bit more on what makes this case unique. In this case, an alternative description could be:

```suggestion
      "description": "no eggs",
```

or:

```suggestion
      "description": "all zeros",
```

---

I would also suggest to make the descriptions about eggs, rather than the mechanics behind it.

---

Descriptions updated